### DISCUSSION TITLE
v5.4.0

### DISCUSSION THREAD

**Discussion by MiguelRipoll23** [MiguelRipoll23/homebridge-securitysystem#157] [General]
This update adds the ability to customize the `Trigger Seconds` when using the `Mode Away Extended` switch.

## Changelog
- Added `Trigger Seconds` option for `Mode Away Extended`.
- Improved setting descriptions.

<hr /><em>This discussion was created from the release <a href='https://github.com/MiguelRipoll23/homebridge-securitysystem/releases/tag/v5.4.0'>v5.4.0</a>.</em>

**Comment by scotthmurray** [comment]
Hi! Just a note to say _thank you_ for creating and continuing to support this very helpful open source project!

**Reply by MiguelRipoll23** [reply]
My pleasure, thank you!
